### ISSUE TITLE
NTR: Hydrops fetalis in newborn offspring (Gestation & perinatal history)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7874]
New term request Hydrops fetalis in newborn offspring (segal_220102154617):

**Comment by pnrobinson** [comment]
@MickeySegal
How is this different from [Hydrops fetalis HP:0001789](https://hpo.jax.org/app/browse/term/HP:0001789)
(Note the HPO does not try to model the family history or PMH in a very detailed way -- this can be done with GA4GH Phenopackets if needed).

**Comment by MickeySegal** [comment]
We have 2 terms:

- Hydrops fetalis in self
- Hydrops fetalis in newborn offspring

Would this be more clear?

- Hydrops fetalis in this individual
- Hydrops fetalis in this individual's newborn offspring

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
HP:4000173
History of previous pregnancy with hydrops fetalis
One or more previous pregnancies in which the fetus developed hydrops fetalis.

Also added History of hydrops fetalis in newborn offspring as a synonym (note alternation to above label!)
